app/scripts/modules/core/src/presentation/refresher/Refresher.tsx

```diff
@@ -39,6 +39,13 @@ export class Refresher extends React.Component<IRefresherProps, IRefresherState>
     });
   }
 
+  public componentWillReceiveProps(nextProps: IRefresherProps): void {
+    if (nextProps.refreshing !== this.props.refreshing ||
+        nextProps.lastRefresh !== this.props.lastRefresh) {
+      this.setState(this.parseState(nextProps));
+    }
+  }
```

Comment:
I think you might also need to add a timer to call this periodically to set the age color...